src/styles/app.scss

```diff
@@ -324,3 +377,37 @@ header h1 {
   }
 }
 
+// Theme switch popup
+// theme selector visible only if JavaScript is available
+
+.theme-container {
+  position: relative;
+}
+
+.theme-icon {
+  display: none;
```

Comment:
It should not be `display: none` by default, only in the `noscript.css` (I commented about it somewhere else).